.github/workflows/wheels-dependencies.sh

```diff
@@ -147,8 +238,17 @@ function build_libavif {
             -DCMAKE_CXX_VISIBILITY_PRESET=hidden \
             -DCMAKE_BUILD_TYPE=$build_type \
             "${libavif_cmake_flags[@]}" \
-            . \
-        && make install)
+            $HOST_CMAKE_FLAGS . )
+
+    if [[ -n "$IOS_SDK" ]]; then
+        # libavif's CMake configuration generates a meson cross file... but it
+        # doesn't work for iOS cross-compilation. Copy in Pillow-generated
+        # meson-cross config to replace the cmake-generated version.
+        cp $WORKDIR/meson-cross.txt $out_dir/crossfile-apple.meson
```

Comment:
At some point in time, would it be possible for you to create an issue/PR with libavif about this?